account_analytic_cost_category/models/account_analytic_line.py

```diff
@@ -9,6 +9,5 @@ class AccountAnalyticLine(models.Model):
     _inherit = "account.analytic.line"
 
     cost_category = fields.Selection(
-        [("cogs", "Cost of Goods Sold"), ("expense", "Expense")],
```

Comment:
What is this? is needed rebase?

Reply:
@AaronHForgeFlow it was showing a warning for that so I remove it, it's on the main branch so rebase wouldn't work, should that be removed on a different pr?

Reply:
@AaronHForgeFlow the warning said that since the field is related there's no need to add the selection options